### ISSUE TITLE
Is ui supposed to to be module.export or corgi?

### ISSUE THREAD

**Issue by agokjr** [yodlr/CoreGI#13]
Is ui supposed to to be coregi aka module.export?

See sample:
```javascript

module.exports = {}
function test(){
	console.log('type:' + typeof(this))
	if(global === this) console.log('global')
	if(module.exports === this) console.log('exports')
}
module.exports.test = test;
test()
console.log('----')
module.exports.test();
```

Are you intending on having ui be short hand for module.exports and coregi?

If also noticed that ui.app was used frequently. Was message chaining avoided?

**Comment by jareddlc** [comment]
yes, I'll be submitting a fix for this soon, and there was no particular reason why chaining was avoided, just code styling.

**Comment by rosskukulinski** [comment]
@agokjr this should be resolved in 0b1a2e1ad6ffa29a8d46d47dddf13f57b5047072

Thanks for opening an issue!
